config.py

```diff
@@ -1969,3 +2018,420 @@ def annotations_to_events(
+    drop_channels_func(raw=raw, subject=subject, session=session)
+    rename_events_func(raw=raw, subject=subject, session=session)
+    fix_stim_artifact_func(raw=raw)
+    find_bad_channels(raw=raw, subject=subject, session=session,
+                      task=get_task(), run=run)
+
+    # Save the data.
+    if save:
+        raw.save(fname=bids_path_out, split_naming='bids', overwrite=True)
+
+    return raw
+
+
+def import_er_data(
+    *,
+    subject: str,
+    session: Optional[str] = None,
+    bads: List[str],
+    save: bool = False
+) -> mne.io.BaseRaw:
+    """Import empty-room data.
+
+    Parameters
+    ----------
+    subject
+        The subject for whom to import the empty-room data.
+    session
+        The session for which to import the empty-room data.
+    bads
+        The selection of bad channels from the corresponding experimental
+        recording.
+    save
+        Whether to save the data to disk or not.
+    """
+    bids_path_er_in = BIDSPath(
+        subject=subject,
+        session=session,
+        run=get_runs(subject=subject)[0],
+        task=get_task(),
+        acquisition=acq,
+        processing=proc,
+        recording=rec,
+        space=space,
+        suffix=get_datatype(),
+        datatype=get_datatype(),
+        root=get_bids_root()
+    ).find_empty_room()
+
+    bids_path_er_out = (bids_path_er_in.copy()
+                        .update(task='noise',
+                                run=None,
+                                suffix='raw',
+                                extension='.fif',
+                                root=get_deriv_root(),
+                                check=False))
+
+    if bads is None:
+        bads = []
+
+    raw_er = load_data(bids_path_er_in)
+    drop_channels_func(raw=raw_er, subject='emptyroom', session=session)
+
+    # Set same set of bads as in the experimental run, but only for MEG
+    # channels (because we won't have any others in empty-room recordings)
+    raw_er.info['bads'] = [ch for ch in bads if ch.startswith('MEG')]
+
+    # Save the data.
+    if save:
+        raw_er.save(fname=bids_path_er_out, split_naming='bids',
+                    overwrite=True)
+
+    return raw_er
+
+
+def get_reference_run_info(
+    *,
+    subject: str,
+    session: Optional[str] = None,
+    run: str
+) -> mne.Info:
+
+    msg = f'Loading info for run: {run}.'
+    logger.info(gen_log_message(message=msg, step=1, subject=subject,
+                                session=session))
+
+    bids_path = BIDSPath(
+        subject=subject,
+        session=session,
+        run=run,
+        task=get_task(),
+        acquisition=acq,
+        recording=rec,
+        space=space,
+        suffix='meg',
+        extension='.fif',
+        datatype=get_datatype(),
+        root=get_bids_root(),
+    )
+
+    info = mne.io.read_info(bids_path)
+    return info
+
+
+# # Leave this here for reference for now
```

Comment:
Keep this part for future reference, I might need to re-use some of these ideas soon